frontend/src/visualizations/CardRenderer.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component, PropTypes } from "react";
+
+import ExplicitSize from "metabase/components/ExplicitSize.jsx";
+
+import { CardRenderer } from '../card/card.charting';
+
+import { getSettingsForVisualization, setLatitudeAndLongitude } from "metabase/lib/visualization_settings";
+
+@ExplicitSize
+export default class CardRendererComponent extends Component {
```

Comment:
The problem was it clashed with `CardRenderer` from `card.charting`. I can figure out something else.